### PR TITLE
docs(widget): partner fee docs update

### PR DESCRIPTION
**Pull request by shoom3301**
# Description

Updated partner fee docs with a preview how it will looks like in CoW Swap UI

### CODE REVIEW

**Comment by elena-zh** [comment]
The build has failed, can't review :(

File: docs/cow-protocol/tutorials/widget.md [hunk 1]
```diff
@@ -59,13 +59,21 @@ import {cowSwapWidget, CowSwapWidgetParams} from '@cowprotocol/widget-lib'
...
+      recipient: '0x0000000000000000000000000000000000000000' // Fee destination address
+  }
 }
 
 cowSwapWidget(widgetContainer, params)
 ```
 
-> **Coming soon, please get in touch to sign up to the beta program [here](https://calendly.com/crystal-cow/cow-swap-widget)**
+The fee in basis points (BPS). One basis point is equivalent to 0.01% (1/100th of a percent).  
+The recipient is the address to which the fee will be sent.
```
Comment (alfetopito) [inline docs/cow-protocol/tutorials/widget.md]: Can we mention it's capped at 1% (100 bps)?

**Review by alfetopito — APPROVED** [review]

File: docs/cow-protocol/tutorials/widget.md (hunk 1 above)
Comment (shoom3301) [reply docs/cow-protocol/tutorials/widget.md]: Nice catch! Added

File: docs/cow-protocol/tutorials/widget.md [hunk 2]
```diff
@@ -59,13 +59,24 @@ import {cowSwapWidget, CowSwapWidgetParams} from '@cowprotocol/widget-lib'
...
+  }
 }
 
 cowSwapWidget(widgetContainer, params)
 ```
 
-> **Coming soon, please get in touch to sign up to the beta program [here](https://calendly.com/crystal-cow/cow-swap-widget)**
+The fee in basis points (BPS). One basis point is equivalent to 0.01% (1/100th of a percent).  
+
+> The fee cannot exceed 1% (100 bps) of the trade amount
```
Comment (alfetopito) [inline docs/cow-protocol/tutorials/widget.md]: I don't think that phrasing is needed.

The BPS is capped at 100 in the setting level, widget users cannot set it higher.

The trade amount/volume fee cap is unrelated to the cap here.

I suggest:

```suggestion
> The fee cannot exceed 1% (100 bps).
```

**Pull request merged**